vendor/protobuf-2.6.1/conanfile.py

```diff
@@ -19,6 +19,10 @@ class ProtobufConan(ConanFile):
         _patch_v2p6p1,
         "CMakeLists.txt",
     ]
+    requires = [
```

Comment:
Shouldn't this be `build_requires`? Thos look like tools just needed for the build of this particular package...

Reply:
That's right, I fixed it.